cmd/lakectl/cmd/root.go

```diff
@@ -208,6 +213,52 @@ func getSyncArgs(args []string, requireRemote bool, considerGitRoot bool) (remot
 	return
 }
 
+func withMessageFlags(cmd *cobra.Command, defaultMsg string, allowEmpty bool) {
+	cmd.Flags().StringP(commitMsgFlagName, "m", defaultMsg, "commit message")
+	cmd.Flags().Bool(allowEmptyMsgFlagName, allowEmpty, "allow an empty commit message")
+}
+
+func withMetadataFlag(cmd *cobra.Command) {
+	cmd.Flags().StringSlice(metaFlagName, []string{}, "key value pair in the form of key=value")
+}
+
+func withCommitFlags(cmd *cobra.Command, defaultMsg string, allowEmptyMessage bool) {
```

Comment:
`allowEmptyMessage` is always false and also not sure we want to allow empty messages for merges at all

Reply:
There's a default merge message which we use when the message is empty (as was the current situation for merge in lakectl and UI)